docs/usage.md

```diff
@@ -52,6 +52,18 @@ to the pytest.
    necessary styles and scripts
 * `--email` - to send the email reports of the test run which was generated
    by --html option. MUST specify --html to send email reports.
+* `--bugzilla` - allows you to skip tests with unresolved bug (test case
+  needs to be properly decorated as you can see in
```

Comment:
If we have the bugzilla mark but dont have the /etc/bz conf file, will it create a problem or skip the test?

Reply:
Just tested this scenario that I moved the configuration file to different location and saw that the test was executed as regularly without any skip and without any error that bugzilla config file is missing.  Anyway you can decide if you pass the --bugzilla parameter to the execution. So if you don't have config file, it doesn't sense to pass --bugzilla cause it has no effect.

You can see also from the code of plugin [here](https://github.com/eanxgeek/pytest_marker_bugzilla/blob/master/pytest_marker_bugzilla.py#L315)
```python
if config.getvalue("bugzilla") and config.getvalue('bugzilla_url'):
```
Which will continue to initialize plugin only if bugzilla parameter provided and bugzilla_url is available.